src/Commands/ScrapViewCommand.php

```diff
@@ -39,7 +39,7 @@ public function handle()
         $name = (string) $this->argument('name');
         $extension = (string) $this->option('extension');
 
-        if (! $this->confirm("Are you sure you want to scrap the view [$name]?")) {
+        if ( ! $this->confirm("Are you sure you want to scrap the view [$name]?")) {
```

Comment:
StyleCI will take care of issues like this, no need to make tiny changes that don't have anything to do with the main feature of this PR.